frontend/ui/widget/dictquicklookup.lua

```diff
@@ -1199,7 +1199,11 @@ function DictQuickLookup:lookupInputWord(hint)
                     end,
                 },
                 {
-                    text = _("Lookup"),
+                    if self.is_wiki then
+                        text = _("Search Wikipedia")
+                    else
+                        text = _("Search dictionary")
+                    end,
```

Comment:
Can't do that, you're in a table list initializer ;).

```lua
text = self.is_wiki and _("Search Wikipedia") or _("Search dictionary"),
```

might work, though ;).

Reply:
Thank you